discord/together.py

```diff
@@ -0,0 +1,12 @@
+class ActivityLink:
+    """A meta class which takes in the payload sent by the API, as the only argument, and generates an
+    invite link from that.
+    
+    BEWARE- THIS CLASS IS NOT INTENTED TO BE USED IN GENERAL AND IS JUST MEANT FOR DOCUMENTATION PURPOSES"""
+
+    def __init__(self, payload: dict):
+        self.code = payload.pop(code)
```

Comment:
`payload.pop(code)` isn't going to work, you forgot the quotes. Also, you don't need `__str__`, because you have `__repr__`. And I don't know if this class is really useful, f-strings allow to form invite links quite easily

Reply:
Ah, yea, my bad. I didn't use VSCode to edit it (my bad). I will open the fork in VSCode from now on!

Reply:
Done This ^^^